confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/macros/ExcerptIncludeMacroConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.confluence.filter.internal.macros;
+
+import org.xwiki.component.annotation.Component;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+/**
+ * Add the page parameter to the excerpt-include macro.
+ *
+ * @version $Id$
+ * @since 9.26.0
+ */
+@Component
+@Singleton
+@Named("excerpt-include")
+public class ExcerptIncludeMacroConverter extends AbstractPageMacroConverter
```

Comment:
Shouldn't you also convert the macro name ? `excerpt-include` definintely does not exist in XWiki.

Reply:
My point is that it does not make sense to implement a partial macro converter, if you can't implement it at this level then just don't.